app/setup.sh

```diff
@@ -1,15 +0,0 @@
-export FLASK_APP="manager.py"
-export FLASK_ENV=development
-export FLASK_RUN_PORT=3000
-
-if [ -f email_config.sh ]; then
```

Comment:
Do we still use these email_ and text_config files?

Reply:
I need to figure out what prod is doing and update this accordingly. probably it'll just be stuff to add to the .env file but that should be documented